FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Tea Jeremic on 19.8.21..
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+internal final class FeedImageMapper {
+	private struct RemoteObject: Decodable {
+		var items: [RemoteFeedImage]
```

Comment:
Suggestion:
This property is never mutate so it can be `let` instead of `var`
```suggestion
		let items: [RemoteFeedImage]
```